xdg.opam

```diff
@@ -22,9 +22,11 @@ build: [
     name
     "-j"
     jobs
+    "--promote-install-files=false"
     "@install"
     "@runtest" {with-test}
     "@doc" {with-doc}
   ]
+  ["dune" "install" "-p" name "--create-install-files" name]
```

Comment:
How did this change sneak in?

Reply:
I've no idea. I'll remove it

Reply:
removed. the diff reappears when I do `./dune.exe runtest --auto-promote`

Reply:
I wonder if it's because your bootstrapped version of dune is old? If not, then we should really fix this issue.